.github/workflows/create-update-pr-env.yml

```diff
@@ -61,7 +61,7 @@ jobs:
         run: |
           cat > .env << EOF
           PUBLIC_URL=${{ env.BRANCH_NAME }} 
-          REACT_APP_API_GRATH=https://api.thegraph.com/subgraphs/name/superfluid-finance/superfluid-matic
```

Comment:
I think we still need the normal graph too , haven't seen PRs reworking all of the queries